yt_dlp/extractor/olympics.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import unified_strdate
+
+
+class OlympicsReplayIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?olympics\.com/tokyo-2020/(?:[a-z]{2}/)?replay/(?P<id>[^/#&?]+)'
```

Comment:
```suggestion
    _VALID_URL = r'(?:https?://)(?:www\.)?olympics\.com/[^/]+]/(?:[a-z]{2}/)?replay/(?P<id>[^/#&?]+)'
```

Reply:
I don't think the site offers older olympics replays 🤔 
And for future ones the url pattern is diff.
Next one is Beijing 2022, imo it's better for it to break and we revisit it in 2022 cuz the site already uses diff way to get videos from the same  tokyo 2020